README.md

```diff
@@ -119,6 +119,11 @@ spec:
 
 ## Release notes
 
+### 3.10.0
+
++ Use `SerializedProcessor` for thread-safe
```

Comment:
Add information for user about fixed problem

Reply:
done